build/read-commit.py

```diff
@@ -0,0 +1,11 @@
+import sys
+import json
+import os
+
+with open('commitinfo.json', 'r') as infile:
```

Comment:
```suggestion
with open(sys.argv[1], 'r') as infile:
```
It looks like you wanted to pass the filename as an argument